packages/components/src/Popover/Popover.mdx

```diff
@@ -0,0 +1,195 @@
+---
+name: Popover
+menu: Components
+route: /components/popover
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Content } from "../Content";
+import { Heading } from "../Heading";
+import { Button } from "../Button";
+import { Checkbox } from "../Checkbox";
+import { useState, useRef } from "react";
+import { Popover } from ".";
+
+# Popover
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Write a description about what Popover component is trying to achieve.
+
+```ts
+import { Popover } from "@jobber/components/Popover";
+```
+
+<Playground>
+  {() => {
+    const divRef = useRef();
+    const [showPopover, setShowPopover] = useState(false);
+    return (
+      <>
+        <span ref={divRef}>
+          <Button
+            label="Toggle Popover"
+            onClick={() => setShowPopover(!showPopover)}
+          />
+        </span>
+        <Popover
+          attachTo={divRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="right"
+          onRequestClose={() => setShowPopover(false)}
+        >
+          <Content>Here is your first popover</Content>
+        </Popover>
+      </>
+    );
+  }}
+</Playground>
+
+<!--
+  It is not necessary to include Design & Usage Guidlelines. Use this section
+  to answer common questions and considerations about component usage.
+-->
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Popover component, consider the
+following:
+
+### Usage
+
+- A popover should be used to introduce a novel UI element or highlight a change
```

Comment:
I realized this is just the content for tooltip moved over - can we reflect the guidelines from the Figma?

> The popover component displays floating informative and actionable content in relation to a target. The popover can contain content, media, or other components. 
> 
> Some scenarios for popover include the following:
> 
> · Show available related actions
> · First time experience
> · To call out product features or changes